shell/components/nav/TopLevelMenu.helper.ts

```diff
@@ -47,10 +47,10 @@ const DEFAULT_SORT: Array<PaginationSort> = [
     asc:   false,
     field: 'spec.internal',
   },
-  // {
-  //   asc:   true,
-  //   field: 'status.conditions[0].status' // Pending API changes https://github.com/rancher/rancher/issues/48092
-  // },
+  {
+    asc:   false,
+    field: 'status.connected'
```

Comment:
The title of this pr seems to think we're supposed to be using `status.condition`

Reply:
i tripped myself up, the correct one is connected. Will update title